package/MDAnalysis/core/selection.py

```diff
@@ -348,7 +363,7 @@ def __init__(self, parser, tokens):
         self.exRadius = float(tokens.popleft())
         self.sel = parser.parse_expression(self.precedence)
 
-    @return_empty_on_apply
+    @return_empty_on_empty_selection
```

Comment:
Rather than replacing `return_empty_on_apply`, you can usually stack decorators like

```
@return_empty_on_apply
@return_empty_on_empty_selection
```

Reply:
Implemented this too!